src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age = 0, sex, isMarried, partnerName } = person;
+  const personPartnerMessage = (sex === 'm')
+    ? `${partnerName} is my wife`
+    : `${partnerName} is my husband`;
+
+  return (
+
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age
```

Comment:
add a line between child elements, like add a line after `h2`

this is needed for better readability